examples/fhir/MN NBS FHIR Order Message.json

```diff
@@ -0,0 +1,773 @@
+{
+  "resourceType": "Bundle",
+  "id": "b4efef3a-749c-457d-956b-568e22768bf3",
+  "identifier": {
+    "value": "b4efef3a-749c-457d-956b-568e22768bf3"
+  },
+  "type": "message",
+  "timestamp": "2023-05-06T05:29:16.000-05:00",
+  "entry": [
```

Comment:
Do we need a total amount of entries, `"entry": 24` ?

Reply:
Looks like total is FHIR search specific